src/App.js

```diff
@@ -1,5 +1,22 @@
+import { useEffect, useState } from 'react';
+import getCharacters from './services/getCharacters';
+import Characters from './components/Characters';
+
 function App() {
-  return <div></div>;
+  const [characters, setCharacters] = useState([]);
+
+  useEffect(() => {
+    getCharacters()
+      .then((characters) => setCharacters(characters))
+      .catch((error) => console.error(error));
```

Comment:
Log out error message

Reply:
I don't agree with this (if you want to log `console.error(error.message)` only).
You will lose information like the error stack, which is very useful to solve issues.

It makes sense to display the `error.message` in the UI to notify the user, because they might not notice the developer console.